src/app/pages/charging-stations/list/charging-stations-list-table-data-source.ts

```diff
@@ -133,6 +133,9 @@ export class ChargingStationsListTableDataSource extends TableDataSource<Chargin
         class: 'col-30p',
         sorted: true,
         direction: 'asc',
+        formatter: (id: string, chargingStation: ChargingStation) => chargingStation.ocpiData
+          && chargingStation.ocpiData.evse && chargingStation.ocpiData.evse.evse_id ?
+          chargingStation.ocpiData.evse.evse_id : id,
```

Comment:
Everything is based on charging station id like filters, ...
If you change the display id of the charging station, it has to be done everywhere in the app.
What do you think?

Reply:
@ellavas Any news on this one ?

Reply:
Absolutely, it must be done everywhere!

Reply:
> Absolutely, it must be done everywhere!

Ok, I'll wait for your change.

Reply:
What is it displayed when evse_id is used versus id?

Reply:
one example: 
=> id = 22740-61273, evse_id => FR*SLF*ESAP*DEV*OCPI*PUBLIC*2*1

This is the more readable field in ocpi data.

Reply:
We keep the id as it is.